sources/packages/backend/apps/api/src/route-controllers/student-scholastic-standings/student-scholastic-standings.institutions.controller.ts

```diff
@@ -196,14 +201,23 @@ export class ScholasticStandingInstitutionsController extends BaseController {
     @UploadedFile() file: Express.Multer.File,
     @Query("validation-only", new DefaultValuePipe(false), ParseBoolPipe)
     validationOnly: boolean,
+    @UserToken() userToken: IInstitutionUserToken,
   ): Promise<PrimaryIdentifierAPIOutDTO[]> {
     // Read the entire file content.
     const fileContent = file.buffer.toString();
+    // Get the file header model.
+    let textHeader: ApplicationBulkWithdrawalHeader;
     // Convert the file raw content into text models.
     let textModels: ApplicationWithdrawalImportTextModel[];
     try {
+      textHeader =
+        this.applicationWithdrawalImportTextService.readText(
```

Comment:
Please avoid calling `this.applicationWithdrawalImportTextService.readText` twice.